frontend/index.js

```diff
@@ -1,5 +1,28 @@
+const todaysDate = new Date()
+
 const dateHeader = document.getElementById('dateHeading');
+const dateToday = document.createTextNode(String(todaysDate.toLocaleString()));
+dateHeader.append(dateToday);
+
+const usersHoursFromGMT = (todaysDate.getTimezoneOffset())/60;
+const usersHoursFromKennedy = usersHoursFromGMT + 5;
+
+const kennedyTimeHeader = document.getElementById('kennedyTime');
+const kennedyTimeZoneOffset = document.createTextNode(usersHoursFromKennedy);
+kennedyTime.append(kennedyTimeZoneOffset);
+
+const hoursAhead = document.createTextNode(' hours ahead of the Kennedy Space Centre');
+const hoursBehind = document.createTextNode(' hours behind the Kennedy Space Centre');
+
+usersHoursFromKennedy > 0 ? kennedyTimeHeader.append(hoursAhead) : kennedyTimeHeader.append(hoursBehind);
+
 
-const dateToday = document.createTextNode(String(new Date()));
+const addCurrentTimeAtKennedySpaceCentre = () => {
+    const kennedyTime = (new Date()).toLocaleTimeString('en-US', {
```

Comment:
Ah, you might have missed Lee and I talking about this yesterday.

The topic of _locales_ is generally for _formatting_ things to look right for the user. So in this case, formatting it to be in `en-US` form makes it look right for an American (you'll find it formats it to show 12-hour clock with AM/PM, whereas without the formatting, it would display in 24-hour clock for us Brits).

So ideally, we'd leave the browser to decide which locale to use for formatting the time, which means not providing `toLocaleTimeString` a locale at all, and letting it use its default value. However, the `timeZone` option you've used is one we still want!

So we want to not provide a locale, but still provide the option for using a time zone.

Reply:
Great. Made those changes. :)